docs/readmes/orc8r/deploy_orcl.md

```diff
@@ -16,11 +16,88 @@ Orcl is the Orchestrator CLI. It is used for managing an Orc8r deployment. It pr
 - Cleanup
 - Debug (perhaps in the future)
 
-![Orcl Big Picture](assets/orc8r/orcl.png)
+## Steps to install Orc8r 1.6 through Orcl
+
+### 1. Build the orc8r_deployer image
+
+```
+git co v1.6
+cd magma/orc8r/cloud/deploy/orc8r_deployer/docker
+
+./run_deployer.bash --deploy-dir <deploy_dir> --build
+
+For e.g:
+./run_deployer.bash --deploy-dir ~/workspace/orc8r_16_deploy_dir --build
+```
+
+### 2. Run Orcl configure and add infra, platform and service level configs
+
+```
+root@ff1b8da0308c:~/project# orcl configure
+Configuring infra deployment variables
+aws_access_key_id(AWS access key id): <aws_access_key_id>
+aws_secret_access_key(AWS access secret): <aws_secret_access_key>
+orc8r_domain_name(Base domain name for AWS Route 53 hosted zone): <orc8r_domain> 
+region(AWS region to deploy Orchestrator components into. The chosen region must provide EKS): us-west-2
+secretsmanager_orc8r_secret(AWS Secret Manager secret to store Orchestrator secrets): <secrets_name>
+
```

Comment:
For Orcl tool, I still expect it to be provided this way. I will clean this up going forward, but currently we need to do this so that we don't fail prechecks.